docs/typescript/workflows.md

```diff
@@ -877,23 +877,20 @@ import PCP from '../content/what-is-a-parent-close-policy.md'
 
 <PCP />
 
-<span id="infinite-workflows" />
+<span id="continueasnew" />
 
-## Entity Workflows
+## `continueAsNew`
```

Comment:
There is now a page that defines Continue-As-New: https://docs.temporal.io/docs/content/what-is-continue-as-new/

Feel free to link to that for source of truth definition.

Reply:
It's linked in the "More info" details